myhpom/static/myhpom/page/_home.scss

```diff
@@ -0,0 +1,296 @@
AAABekYmDhGThAAAAAADYyzIIkpGFAwAAAADgCM8gSEQWDgAAAACAo2TiIAFZOAAAAAAAzrIMguBk4QAAAAAAuMIzCAKThQMAAAAA4CqZOAhIFg4AAAAAgFEsgyAYWTgAAAAAAEbyDIJAZOEAAAAAABhNJg4CkIUDAAAAAOAulkGwmCwcAAAAAAB38gyChWThAAAAAAC4m0wcLCALBwAAAADALJZBMJksHAAAAAAAM3kGwUSycAAAAAAAzCYTBxPIwgEAAAAAsIplENxMFg4AAAAAgJU8g+BGsnAAAAAAAKwmEwc3kIUDAAAAACAKyyAYTBYOAAAAAIBIPINgIFk4AAAAAACikYmDAWThAAAAAACIyjIILpKFAwAAAAAgMs8guEAWDgAAAACA6GTi4ARZOAAAAAAAsrAMgoNk4QAAAAAAyMQzCA6QhQMAAAAAIBuZONhBFg4AAAAAgKwsg+AFWTgAAAAAADLzDIInZOEAAAAAAMhOJg6+IAsHAAAAAEAVlkHwgSwcAAAAAACVeAbBG7JwAAAAAABUIxMHmywcAAAAAAB1WQbRniwcAAAAAACVeQbRmiwcAAAAAADVycTRkiwcAAAAAABdWAbRjiwcAAAAAACdeAbRiiwcAAAAAADdyMTRgiwcAAAAAABdWQZRniwcAAAAAACdeQZRmiwcAAAAAADdycRRkiwcAAAAAAD8ZBlEObJwAAAAAADwh2cQpcjCAQAAAADAezJxlCALBwAAAAAAX7MMIj1ZOAAAAAAAeMwziNRk4QAAAAAA4DmZOFKShQMAAAAAgH0sg0hHFg4AAAAAAPbzDCIVWTgAAAAAADhGJo4UZOEAAAAAAOAcyyDCk4UDAAAAAIDzPIMITRYOAAAAAACukYkjJFk4AAAAAAAYwzKIcGThAAAAAABgHM8gQpGFAwAAAACAsWTiCEEWDgAAAAAA7mEZxHKycAAAAAAAcB/PIJaShQMAAAAAgHvJxLGELBwAAAAAAMxhGcR0snAAAAAAADCPZxBTycIBAAAAAMBcMnFMIQsHAAAAAABrWAZxO1k4AAAAAABYxzOIW8nCAQAAAADAWjJx3EIWDgAAAAAAYrAMYjhZOAAAAAAAiMMziKFk4QAAAAAAIBaZOIaQhQMAAAAAgJgsg7hMFg4AAAAAAOLyDOISWTgAAAAAAIhNJo5TZOEAAAAAACAHyyAOk4UDAAAAAIA8PIM4RBYOAAAAAABykYljF1k4AAAAAADIyTKIl2ThAAAAAAAgL88gnpKFAwAAAACA3GTi+JIsHAAAAAAA1GAZxCeycAAAAAAAUIdnEO/IwgEAAAAAQC0ycWzbJgsHAAAAAABVWQYhCwcAAAAAAIV5BjUnCwcAAAAAALXJxDUlCwcAAAAAAD1YBjUkCwcAAAAAAH14BjUjCwcAAAAAAL3IxDUhCwcAAAAAAD1ZBjUgCwcAAAAAAH15BhUnCwcAAAAAAL3JxBUlCwcAAAAAAGybZVBJsnAAAAAAAMBvnkHFyMIBAAAAAABvycQVIQsHAAAAAAB8xTKoAFk4AAAAAADgEc+g5GThAAAAAACAZ2TikpKFAwAAAAAA9rAMSkgWDgAAAAAA2MszKBlZOAAAAAAA4AiZuCRk4QAAAAAAgDMsgxKQhQMAAAAAAM7yDApOFg4AAAAAALhCJi4oWTgAAAAAAGAEy6CAZOEAAAAAAIBRPIOCkYUDAAAAAABGkokLQhYOAAAAAAC4g2VQALJwAAAAAADAXTyDFpOFAwAAAAAA7iQTt4gsHAAAAAAAMINl0AKycAAAAAAAwCyeQZPJwgEAAAAAADPJxE0iCwcAAAAAAKxgGTSBLBwAAAAAALCKZ9DNZOEAAAAAAICVZOJuIgsHAAAAAABEYBl0A1k4AAAAAAAgCs+gwWThAAAAAACASGTiBpGFAwAAAAAAIrIMGkAWDgAAAAAAiMoz6CJZOAAAAAAAIDKZuJNk4QAAAAAAgAwsg06QhQMAAAAAALLwDDpIFg4AAAAAAMhEJm4nWTgAAAAAACAjy6AdZOEAAAAAAICsPINekIUDAAAAAAAyk4l7QBYOAAAAAACowDLoC7JwAAAAAABAFZ5BH8jCAQAAAAAAlcjE/SILBwAAAAAAVGQZtMnCAQAAAAAAdbV/BsnCAQAAAAAAlbXNxMnCAQAAAAAAHbRcBsnCAQAAAAAAXbR7BsnCAQAAAAAAnbTJxMnCAQAAAAAAHbVYBsnCAQAAAAAAXZV/BsnCAQAAAAAAnZXNxMnCAQAAAAAAFF0GycIBAAAAAAD8VO4ZJAsHAAAAAADwR5lMnCwcAAAAAADAZyWWQbJwAAAAAADA/7dzBzcMwzAQBAP3X3R++cSGZVkSSWGmjcMt58qPQbJwAAAAAAAA18pm4mThAAAAAAAA7pV8BsnCAQAAAAAAtCk3BsnCAQAAAAAAtCuTiZOFAwAAAAAAeK7EM0gWDgAAAAAAoE/6MUgWDgAAAAAAoF/aTJwsHAAAAAAAwHspn0GycAAAAAAAAGOkG4Nk4QAAAAAAAMZJk4mThQMAAAAAABgvxTNIFg4AAAAAAGCO8DFIFg4AAAAAAGCesEycLBwAAAAAAMB8Ic8gWTgAAAAAAIA1lo9BsnAAAAAAAADrLMvEycIBAAAAAACst+QZJAsHAAAAAAAQY/oYJAsHAAAAAAAQZ1omThYOAAAAAAAg3pRnkCwcAAAAAABADsPHIFk4AAAAAACAPIZl4mThAAAAAAAA8hnyDJKFAwAAAAAAyOn1GCQLBwAAAAAAkFd3Jk4WDgAAAAAAIL+uZ5AsHAAAAAAAQA2PxyBZOAAAAAAAgDqaM3GycAAAAAAAAPU0PYNk4QAAAAAAAGq6HYNk4QAAAAAAAOq6zMTJwgEAAAAAANR3+gyShQMAAAAAANjD3xgkCwcAAAAAALCPXyZOFg4AAAAAAGA/x+cjCwcAAAAAALCrQxYOAAAAAABgX1/MUmMQj/2UyQAAAABJRU5ErkJggg==');
```

Comment:
A comment on what this binary blob is and why its there would be heplful for posterity.